tests/sampler/test_parscan.cpp

```diff
@@ -0,0 +1,83 @@
+#include <iostream>
+#include <sstream>
+#include <vector>
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+#include "Kokkos_Core.hpp"
+
+using ::testing::HasSubstr;
+using ::testing::Not;
+
+struct Tester {
+  template <typename execution_space>
+  explicit Tester(const execution_space& space) {
+    //! Explicitly launch a kernel with a name, and run it 150 times with kernel
+    //! logger. Use a periodic sampling with skip rate 51. This should print
+    //! out 2 invocations, and there is a single matcher with a regular
+    //! expression to check this.
+
+    long int N = 1024;
+    int64_t result;
```

Comment:
Won't compile on some platforms...
```suggestion
    long int result;
```
You must match the functor argument that is passed by reference.

Reply:
Thanks for that. It is fixed.